modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -1275,46 +1658,107 @@ Update the option `indexProvider` with the value `text-2.0`.
 ----
 CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-2.0'}
 ----
-====
 
-.Using differently ordered return items in a `UNION` clause
-====
+======
+[.include-with-GQLSTATUS-code]
+======
 Query::
 +
 [source,cypher]
 ----
-RETURN 'val' as one, 'val' as two
-UNION
-RETURN 'val' as two, 'val' as one
+CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-1.0'}
+----
+
+Returned GQLSTATUS code::
+01N01
+
+Returned Status Description::
+warn: feature deprecated with replacement.
+`text-1.0` is deprecated.
+It is replaced by `text-2.0`.
+
+Suggestions for improvement::
+Update the option `indexProvider` with the value `text-2.0`.
++
+[source,cypher]
+----
+CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-2.0'}
+----
+======
+=====
+
+
+
+===== Using a renamed or a deprecated procedure
+
+[.tabbed-example]
+=====
+[.include-with-neo4j-code]
+======
+
+[source,cypher]
+----
+CALL unsupported.dbms.shutdown
 ----
 
 Description of the returned code::
-All subqueries in a UNION [ALL] should have the same ordering for the return columns.
-Using differently ordered return items in a UNION [ALL] clause is deprecated and will be removed in a future version.
+The query used a deprecated procedure: `'unsupported.dbms.shutdown'`.
 
 Suggestions for improvement::
-Use the same order for the return columns in all subqueries combined by a `UNION` clause.
+Remove the use of the deprecated procedure.
+If there is a suggested replacement, update to use the replacement instead.
 
-+
+======
+[.include-with-GQLSTATUS-deprecated-with-replacement]
+======
 [source,cypher]
 ----
-RETURN 'val' as one, 'val' as two
-UNION
-RETURN 'val' as one, 'val' as two
+CALL unsupported.dbms.shutdown
 ----
-====
 
-.Using id() function
-====
+Returned GQLSTATUS code::
+01N01
+
+Returned Status Description::
+warn: feature deprecated with replacement.
+`unsupported.dbms.shutdown` is deprecated.
```

Comment:
I have a question here. We use non-existing procedure in this example, but in the next example we're showing a real use case with `id()` function which has been replaced with `elementid()`.
So my question is: will it be better to change `unsupported.dbms.shutdown` to (for ex.) `cdc.query`, the real deprecated procedure? In this case, we can say that `cdc.query` was deprecated and replaced by `db.cdc.query`.  Will it make the example more visual and helpful?

Reply:
Yes, absolutely, a real example is better. I think at the time we wrote these docs, we did not have any built-in procedure which was actually deprecated so therefore we made one up. Feel free to update the example